hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/storage/ContainerProtocolCalls.java

```diff
@@ -470,7 +472,8 @@ public static XceiverClientReply writeChunkAsync(
       builder.setEncodedToken(tokenString);
     }
     ContainerCommandRequestProto request = builder.build();
-    return xceiverClient.sendCommandAsync(request);
+//    return xceiverClient.sendCommandAsync(request, ReplicationLevel.MAJORITY_COMMITTED);
+    return xceiverClient.sendCommandAsync(request, ReplicationLevel.ALL_COMMITTED);
```

Comment:
if WriteChunk is MAJORITY_COMMITTED or ALL_COMMITTED, does it mean waitOnFlushFutures is no longer needed too?